Packs/Phishing/Wizards/wizard-Phishing.json

```diff
@@ -0,0 +1,585 @@
+{
+    "id": "Phishing",
+    "version": -1,
+    "modified": "2022-04-10T11:55:54.250933+03:00",
+    "fromVersion": "6.9.0",
+    "name": "Phishing",
+    "description": "Welcome to the Deployment Wizard!  These steps will guide you through configuring your content pack so you'll have a working use case when the wizard finishes. Make sure to follow the steps in order, you can always continue the wizard where you left off. Learn more about this pack’s deployment.",
+    "dependency_packs": [
+      {
+        "name": "Email Gateways",
+        "min_required": 1,
+        "packs": [
+          {
+            "name": "EWS",
+            "display_name": "EWS Mail (Note: Please install also the EWS Mail Sender Pack)"
```

Comment:
Recommend:
 "display_name": "EWS Mail (Make sure you also install the EWS Mail Sender pack.)"

Reply:
Done